packages/veritone-json-schemas/schemas/vtn-standard/master.json

```diff
@@ -0,0 +1,223 @@
+        "schemaId": {
+          "type": "string",
+          "description": "vtn-standard.master.schema.json or subschema.schema.json (provided by Veritone)",
+          "example": "https://docs.veritone.com/schemas/vtn-standard/master.json"
+        },
+        "sourceEngineId": {
+          "type": "string",
+          "description": "<GUID> of the engine (provided by Veritone)",
+          "example": "fe261482-af8d-4709-aefb-ff1c19b61eb9"
+        },
+        "sourceEngineName": {
+          "type": "string",
+          "description": "Engine name used to generate output (provided by Veritone) // TO DO: Verify if this is internal or external engine name",
+          "example": "capio"
+        },
+        "taskPayload": {
+          "type": "object",
+          "description": "JSON key/value task payload describing the tasks used to start the engine run (provided by Veritone) // TO DO: LINK TO DOCS EXPLAINING TASK PAYLOAD",
+          "example": {
+            "detectionThreshold": 0.7,
+            "processingFPS": 30
+          }
+        },
+        "taskId": {
+          "type": "string",
+          "description": "<GUID> of the task (provided by Veritone)",
+          "example": "6b9cd86e-de5c-4795-b605-a9b3b29fa3cf"
+        },
+        "generatedDateUTC": {
+          "type": "string",
+          "description": "DateTime (ISO8601) of when the engine result was generated (provided by Veritone if not included)",
+          "format": "date-time",
+          "example": "2017-12-08T17:19:02Z"
+        },
+        "externalSourceId": {
+          "type": "string",
+          "description": "Vendor specific referenceId to this engine result (optional)",
+          "example": "1"
+        },
+        "validationContracts": {
+          "type": "array",
+          "items": [
+            {
+              "type": "string",
+              "enum": [
+                "audio-fingerprint",
+                "ocr",
+                "face",
+                "object",
+                "transcript"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "confidence": {
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "description": "Confidence of the result expressed as a range of 0.0 to 1.0"
+    },
+    "objectCategory": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "class": {
+            "type": "string",
+            "description": "The class of the object recognized",
+            "example": "For example, if the label of an object was dog, the class of the object could be animal"
+          },
+          "@id": {
+            "type": "string",
+            "description": "Private id variable used by the engine to refer to its internal dataset",
+            "example": "1"
+          }
+        },
+        "required": [
+          "class"
+        ]
+      },
+      "description": "Optional category for object recognition"
```

Comment:
Optional categor**ies** for object recognition. Used for referencing external taxonomies.